src/test/java/org/springframework/integration/aws/inbound/KinesisMessageDrivenChannelAdapterTests.java

```diff
@@ -236,16 +236,17 @@ void testKinesisMessageDrivenChannelAdapter() {
 		assertThat(message.getPayload()).isInstanceOf(List.class);
 		messagePayload = (List<String>) message.getPayload();
 		assertThat(messagePayload).size().isEqualTo(2);
-		assertThat(messagePayload).contains("bar");
-		assertThat(messagePayload).contains("foobar");
+		assertThat(messagePayload)
+				.contains("bar")
+				.contains("foobar");
```

Comment:
This one does not look like best practice.
Why don't use this code then:
```
	assertThat(messagePayload).contains("bar", "foobar");
```
?
Was this automated?
Would you mind double checking the change before pushing to PR?

Reply:
Will apply the change. That indeed looks even better 👍🏻. And will write it down to create a rule for that, such that we can automate this change as well.

Double checked, but simply didn't think of that one.